### DISCUSSION TITLE
Option to provide vector of file names to read.files()

### DISCUSSION THREAD

**Discussion by hrj21** [ImmuneDynamics/Spectre#84] [! News]
Hi Spectr-acular people!

I was wondering if you think it could be useful and/possible to supply a vector of filenames, rather than just the directory name, to the `read.files()` function. For example, in my current project I have a large number of  .fcs files that are each quite big. I don't want to have to move these around on disk so the directory contains only the ones I want to read, and I don't want to read them all into my R session. I would prefer to create a character vector of the files I'm specifically interested in, and have `read.files()` read in only these files. Would anyone else find that useful?

Best wishes
Hefin

**Comment by tomashhurst** [comment]
@hrj21 I was just thinking about how helpful that would be in the last couple of weeks! I'll add it to the next release!
